tools/configAuth.js

```diff
@@ -1,3 +1,4 @@
+// This isn't safe - let's move this to an environment variable
```

Comment:
Yup - instead of putting your secret on a public github file (configAuth.js), you could put it in an environment variable the way we manage PORT and MONGODB_URI in the srcServer.js file.